web/packages/teleport/src/Player/DesktopPlayer.tsx

```diff
@@ -175,6 +177,14 @@ const useDesktopPlayer = ({ clusterId, sid }) => {
     [setPlayerStatus, setStatusText]
   );
 
+  const clientOnTdpInfo = useCallback(
+    (info: string) => {
+      setPlayerStatus(StatusEnum.COMPLETE);
+      setStatusText(info);
+    },
+    [setStatusText]
```

Comment:
Why do we include `setStatuxText` but not `setPlayerStatus`?

In any case, we can probably omit both. The React hooks docs say:
> React guarante
es that setState function identity is stable and won’t change on re-renders. This is why it’s safe to omit from the useEffect or useCallback dependency list.

Reply:
c4c4aa2c6e2cad87db660351b95cc66391bdaf46